src/main/java/org/openmrs/maven/plugins/Setup.java

```diff
@@ -183,149 +174,94 @@ public Setup(MavenProject mavenProject,
      * @throws MojoExecutionException
      */
     public String setup(Server server, boolean isCreatePlatform, boolean isCopyDependencies) throws MojoExecutionException, MojoFailureException {
+
         versionsHelper = new VersionsHelper(artifactFactory, mavenProject, mavenSession, artifactMetadataSource);
+        moduleManager = new ModuleManager(mavenProject, mavenSession, pluginManager, versionsHelper);
```

Comment:
Inject as a component same as Wizard. You could make the VersionsHelper a component as well.